src/components/ZCLToolbar.vue

```diff
@@ -250,6 +264,11 @@ export default {
         )
       }
     },
+    saveChanges() {
```

Comment:
this is already done in menu.js

Reply:
These changes are done in order to embed ZAP into Studio v6, where there's no electron menu...